protocol/communities_messenger_test.go

```diff
@@ -1188,45 +1188,65 @@ func (s *MessengerCommunitiesSuite) TestSyncCommunity() {
 
 func (s *MessengerCommunitiesSuite) TestSyncAccounts() {
 
-	mainAccount := []accounts.Account{
-		{Address: types.Address{0x02}, Wallet: true, Chat: true},
+	mainAccount := accounts.Account{
+		Address: types.Address{0x01},
+		Wallet: true,
+		Chat: true,
 	}
-
-	watchOnly1 := []accounts.Account{
-		{Address: types.Address{0x01}, Name: "Alice watch only", Color: "green", Type: "watch"},
+	watchOnly1 := accounts.Account{
+		Address: types.Address{0x02},
+		Name: "Alice watch only",
+		Color: "green",
+		Type: accounts.AccountTypeWatch,
 	}
-
-	watchOnly2 := []accounts.Account{
-		{Address: types.Address{0x01}, Name: "Alice watch only 2", Color: "blue", Type: "watch"},
+	watchOnly2 := accounts.Account{
+		Address: types.Address{0x03},
+		Name: "Alice watch only 2",
+		Color: "blue",
+		Type: accounts.AccountTypeWatch,
 	}
 
-	// Create a main acount on alice
-	s.NoError(s.alice.settings.SaveAccounts(mainAccount))
+	// Create a main account on alice
+	s.NoError(s.alice.settings.SaveAccounts([]accounts.Account{mainAccount}))
```

Comment:
is this change a code pattern related one or does this change the behaviour itself? I see you passed [] directly here instead of on the variable.

Reply:
Yeah, this is just a code pattern change, and mainly so that we can compare the values of the accounts to the values taken from the db. Your approach was 100% valid, and using your approach you could chose to compare the values like:

```go
// Check account values match the expected values
	s.Require().Equal(mainAccount[0].Address, acc1[0].Address)
	s.Require().Equal(mainAccount[0].Name, acc1[0].Name)
	s.Require().Equal(mainAccount[0].Color, acc1[0].Color)
	s.Require().Equal(mainAccount[0].Type, acc1[0].Type)
```

It is an awkward setup because the main function `SaveAccounts` expects an array of Account, but we just want to add one at a time.